packages/website/package.json

```
@@ -9,14 +14,19 @@
     "start": "waku start --with-ssr"
   },
   "dependencies": {
-    "autoprefixer": "^10.4.16",
-    "bright": "^0.8.4",
     "express": "^4.18.2",
-    "glob": "^10.3.10",
     "react": "18.3.0-canary-a41957507-20231017",
     "react-dom": "18.3.0-canary-a41957507-20231017",
     "react-server-dom-webpack": "18.3.0-canary-a41957507-20231017",
-    "tailwindcss": "^3.3.3",
-    "waku": "workspace:*"
+    "waku": "0.16.0"
```

```suggestion
    "waku": "workspace:*"
```
We use the workspace protocol in monorepo.